IBM/terms.yml

```diff
@@ -0,0 +1,210 @@
+extends: substitution
+message: Consider using '%s' instead of '%s'
+link: 'https://www.ibm.com/developerworks/library/styleguidelines/index.html#wordlist'
+level: error
+ignorecase: true
+# swap maps tokens in form of bad: good
+swap:
+  a lot: many or much
+  a number of: several
+  abort: cancel or stop
+  administrate: administer
+  all caps: uppercase
+  and/or: a or b or a b or both
+  as long as: if or provided that
+  as per: according to or as or as in
+  as well as: and
+  backlevel: earlier or previous or not at the latest level
+  back-level: earlier or previous or not at the latest level
+  backward compatible: compatible with earlier versions
+  backward-compatible: compatible with earlier versions
+  Big Blue: IBM
+  blink: flash
+  blue screen of death: stop error
+  bottom right: lower right or lower-right
+  bottom-right: lower right or lower-right
+  bottom left: lower left or lower-left
+  bottom-left: lower left or lower-left
+  breadcrumbing: breadcrumb trail
+  canned: preplanned or preconfigured or predefined
+  case insensitive: not case-sensitive
+  catastrophic error: unrecoverable error
+  CBE: Common Base Event
+  CBTS: CICS BTS or BTS
+  cold boot: hardware restart
+  cold start: hardware restart
+  comes with: includes
+  componentization: component-based development or component model or component architecture or shared components
+  componentize: develop components
+  comprised of: consist of
+  connect with: connect to
+  context menu: menu or pop-up menu
+  contextual help: help or context-sensitive help
+  control-click: press Ctrl and click
+  crash: fail or lock up or stop or stop responding
```

Comment:
Instead of using 'or, ... or, ...', you can write the options as pipe-delimited (`fail|lock up|stop`) and Vale will convert it to a sentence in its output.

(This also makes the rule compatible with Vale Server's in-editor suggestions -- a UI drop-down with be populated with each option instead of only the 'or, ... or, ...' string.)